test/integral.jl

```diff
@@ -0,0 +1,31 @@
+using Symbolics, Test
+using DomainSets
+@variables x y
+
+I1 = Integral(x, ClosedInterval(1, 5))
+I2 = Integral(x, ClosedInterval(1, 5))
+@test I1 == I2
+
+@variables v(..) u(..) x y
+D = Differential(x)
+Dxx = Differential(x)^2
+I = Integral(y, ClosedInterval(1, 5))
+eq = D((I(u(x,y)))) ~ 0
+eq_test = I(D(u(x,y)))
+@test isequal(expand_derivatives(eq.lhs), Symbolics.value(eq_test))
+
+eq = Dxx((I(u(x,y)))) + I(D(u(x,y))) ~ 0
+eq_test = I(D(u(x,y))) + I(D(D(u(x,y))))
+@test isequal(expand_derivatives(eq.lhs), Symbolics.value(eq_test))
+
+I = Integral(y, ClosedInterval(1, v(x)))
+eq = D((I(u(x,y)))) ~ 0
+eq_test = I(D(u(x,y))) + D(v(x))*u(x, v(x))
+@test isequal(expand_derivatives(eq.lhs), Symbolics.value(eq_test))
+
+I = Integral(y, ClosedInterval(1, v(x)))
```

Comment:
What if you test an integral with the integrand `u^2`? Does it expand the derivative?

Reply:
Yes, it does expand. Should I add this as a test?

Reply:
Yes

Reply:
I did, in the last commit.